### ISSUE TITLE
Import a package that does not exist

### ISSUE THREAD

**Issue by rose-tmp** [prometheus/prometheus#8907]
The client package imported by the code on [line 25](https://github.com/prometheus/prometheus/blob/main/documentation/examples/remote_storage/remote_storage_adapter/influxdb/client.go#L25) no longer exists in influxdb of version [v2.0.7](https://github.com/influxdata/influxdb/releases/tag/v2.0.7) (the latest version). It still exists in its last [v1.9.1](https://github.com/influxdata/influxdb/releases/tag/v1.9.1) .So should we update the code of this module synchronously?

**Comment by roidelapluie** [comment]
Yes, if the example keeps working. Are you willing to contribute this? Thanks!

**Comment by slashpai** [comment]
Can I work on this? Any starting point would be helpful :)

**Comment by codeWithUtkarsh** [comment]
I see that this issue is still open. I am an active contributor to the open-source community. Please acknowledge me so that I can start working on this.

**Comment by jackwener** [comment]
Maybe I can try to fix it.

**Comment by beorn7** [comment]
Hello from the bug scrub!

Sorry for the long radio silence. Yes, the example code still needs to be fixed. Should be easy. Whoever wants to do it, please do. And yes, we'll try to have a look on this issue and assign it to whoever wants to work on it.

**Comment by KofClubs** [comment]
This issue shall be fixed by #15802 

(Finally some free time found from my busy work😊)
